addons/mail/static/src/models/popover_view.js

```diff
@@ -106,7 +106,7 @@ registerModel({
                 return 'ChannelInvitationForm';
             }
             if (this.emojiPickerView) {
-                return 'EmojiPicker';
+                return 'EmojiPickerView';
```

Comment:
to remember for later: link between component name and model name should be made in modelling so that we don't have to hard-code this here